test/codegen/enum.c

```diff
@@ -0,0 +1,43 @@
+enum {
+  TRUE,
+  FALSE,
+};
+
+int main() {
+  // Convertible to int.
+  int f = FALSE;
+  __builtin_print(f);
```

Comment:
nit: Since f is declared from FALSE, I assume it was 0 at first. But it is 1 due to the sequence of enum. 🤣 Consider swapping the order of TRUE, FALSE.

Reply:
I swapped the order and initialized `f` with `TRUE` instead, so that the expected output isn't changed. :love_you_gesture: